README.md

```diff
@@ -1,127 +1,71 @@
+    testImplementation 'junit:junit:4.13'
+    ohosTestImplementation 'com.huawei.ohos.testkit:runner:1.0.0.100'
+}
+```
+2. For using verificationview in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
+```groovy
+	dependencies {
+		implementation fileTree(dir: 'libs', include: ['*.har'])
+		testImplementation 'junit:junit:4.13'
+	}
+```
 
-# 三、对应方法的说明
-
-|                    方法                    |             说明             |
-| :--------------------------------------: | :------------------------: |
-| setShowInterferenceLines(boolean showInterferenceLines) |      设置是否显示干扰线条，默认显示       |
-| setShowInterferenceCircles(boolean showInterferenceCircles) |      设置是否显示干扰圆点，默认显示       |
-|      setTextBold(boolean textBold)       |     设置验证码文本是否加粗，默认不加粗      |
-| setCircleColorRandom(boolean circleColorRandom) |    设置干扰圆点是否为随机颜色，默认为随机     |
-| setLineColorRandom(boolean lineColorRandom) |    设置干扰线条颜色是否为随机，默认为随机     |
-| setVerificationText(String verificationText) |         设置验证码文本内容          |
-| setVerificationCodeBackground(int verificationCodeBackground) |      设置验证码的背景色，默认为灰色       |
-| setInterferenceLinesCount(int interferenceLinesCount) |      设置干扰线条的数量，默认为10条      |
-| setInterferenceLinesColor(int interferenceLinesColor) |  设置干扰线条的颜色，如果设置了颜色，则颜色不再   |
-| setInterferenceLinesWidth(float interferenceLinesWidth) |       设置干扰线条的宽度，默认为3       |
-| setInterferenceCirclesCount(int interferenceCirclesCount) |         设置干扰圆点的数量          |
-| setInterferenceCirclesColor(int interferenceCirclesColor) |    设置干扰圆点的颜色，如果设置，则不再随机    |
-| setInterferenceCirclesRadius(float interferenceCirclesRadius) |       设置干扰圆点的半径，默认为5       |
-|       setTextColor(int textColor)        |        设置验证码文本，默认黑色        |
-|        setTextSize(int textSize)         |         设置验证码文本大小          |
-|     setUnderLine(boolean underLine)      |       设置验证码文本是否有下划线        |
-|      setTextSkewX(float textSkewX)       | 设置验证码文本的倾斜值，正数左斜，负数右斜，默认为0 |
-|    setStrokeWidth(float strokeWidth)     |         设置验证码文本的宽度         |
-
-
-
-# 四、效果图
-
-![效果图](https://ww1.sinaimg.cn/large/006y8lValy1fbxzq88636g309t0injss.gif)
-
-
-# 五、联系方式
+## Usage
+### Include following code in your layout
+``` xml
+<?xml version="1.0" encoding="utf-8"?>
+ <com.zhangym.customview.VerificationCodeView
+      ohos:id="$+id:verificationCodeView"
+      ohos:width="0vp"
+      ohos:height="50vp"
+      ohos:weight="1"
+      zhangym:interferenceCirclesCount="40"
+      zhangym:interferenceCirclesRadius="5"
+      zhangym:interferenceLinesCount="8"
+      zhangym:isShowInterferenceCircles="true"
+      zhangym:isShowInterferenceLines="true"
+      zhangym:isTextBold="true"
+      zhangym:textSize="26fp"
+      zhangym:textSkewX="0"
+      zhangym:verificationText="9D27" />
+```
 
-- [我的博客](http://zhangym.xyz/)
-- **Gmail**:yongming9011@gmail.com
+### Include following code in your activity
```

Comment:
Describe all the custom attributes / setter function of custom attributes(like in base library) - give a one line description